plugins/flashrom/fu-flashrom-internal-device.c

```diff
@@ -105,6 +106,18 @@ fu_flashrom_internal_device_write_firmware(FuDevice *device,
 	g_autoptr(GBytes) blob_fw = fu_firmware_get_bytes(firmware, error);
 	if (blob_fw == NULL)
 		return FALSE;
+		
+	/* Check if CMOS needs a reset */
+	if (fu_device_has_private_flag(device, FU_FLASHROM_DEVICE_FLAG_RESET_CMOS)) {
+		g_debug("Attempting CMOS Reset");		
+		if (fu_reset_cmos(NULL)) {
```

Comment:
just pass the error in rather than NULL, and then you don't need to do a g_set_error_literal at all